### PR TITLE
minimal example to run jitsi behind traefik

### PR DESCRIPTION
**Pull request by SebastianKobs**
Hi, 
adding to my comment in https://github.com/jitsi/docker-jitsi-meet/issues/220
I've added an minmal example for traefik.
Kind regards
Seb

### CODE REVIEW

File: examples/traefik/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Basic configuration to use with the traefik reverse proxy
+
+Note: Tested with traefik 1.7
+
+- Remove the port-binds for the web service
+- The provided example assumes an external network with the name "web"
+- I've added comments where to adjust the docker-compose.yml, but I assume that you know your stuff, if you're running traefik.
```
Comment (funkyfuture) [inline examples/traefik/README.md]: this is not appropriate for a documentation.

File: examples/traefik/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Uncomment and set DOCKER_HOST_ADDRESS in .env  
+Im pretty sure, that this is mandatory for the docker-setup and should be clearer in the original README. Could be the proxying, didn't investigate further.
```
Comment (funkyfuture) [inline examples/traefik/README.md]: do not guess what is needed and / or correct!

Comment (SebastianKobs) [reply examples/traefik/README.md]: You're right on this. Didn't work for me without it, doesn't mean it's mandatory. Will change that.

File: examples/traefik/README.md (hunk 1 above)
Comment (SebastianKobs) [reply examples/traefik/README.md]: I've assumed(wrongly?) that people who wants to run jitsi behind traefik, know a bit about setting it up and just added the steps which I've done. I can write a complete setup-guide including setting up traefik from the getgo, if you consider this more helpful. Not sure, if the traefik-setup-part would be out of scope?

**Comment by SebastianKobs** [comment]
>  ... move the exposure of web ports (as defined now in `docker-compose.yml`) to a separate compose file, i have no idea whether the maintainers would accept that.

You can leave them in, but I consider it a better practice to not expose them on the host, unless absolutely needed. As for maintaining it, I'll happily update the example when needed, but will see if I can come up with a better way. Thanks for your helpfull comments.

File: examples/traefik/README.md (hunk 1 above)
Comment (funkyfuture) [reply examples/traefik/README.md]: the point is that you're writing in the singular first person, unlike the rest of the document. also punctuation is missing, there are unnecessary line breaks in the markup. just use a hyperlink to reference traefik, don't point out the obvious ("you know your stuff, if …")

Comment (funkyfuture) [reply examples/traefik/README.md]: yes, an introduction to traefik would be out of scope.

**Comment by funkyfuture** [comment]
exposing unneeded ports to the public is not a good idea. it might cause confusion and makes them unavailable for other purposes.

are you aware of this Docker-Compose feature: https://docs.docker.com/compose/extends/ ?

**Comment by SebastianKobs** [comment]
Yes I'm aware of it. Could use that, but this example stems from my own setup. I changed the wording in the README to include your suggestions,.

**Comment by saghul** [comment]
Cheers!

**Pull request merged**